src/script/components/calling/VideoControls/EmojisBar/EmojisBar.styles.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Wire
+ * Copyright (C) 2020 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {CSSObject} from '@emotion/react';
+
+import {media} from '@wireapp/react-ui-kit';
+
+export const wrapperStyles: CSSObject = {
```

Comment:
since all other wrapperStyles have a part of the name that indicates WHICH part of the app they should apply to, i would like to see this here as well
```suggestion
export const emojisBarWrapperStyles: CSSObject = {
```